lib/_sendEvent.ts

```diff
@@ -15,25 +16,27 @@ export function makeSendEvents(requestFn: RequestFnType) {
       );
     }
 
-    const events: InsightsEvent[] = eventData.map(data => {
-      const {filters, ...rest}  = data;
+    const events: InsightsEvent[] = eventData.map((data) => {
+      const { filters, ...rest } = data;
+
       const payload: InsightsEvent = {
         ...rest,
         userToken: data?.userToken ?? this._userToken
       };
       if (!isUndefined(filters)) {
-        payload.filters = filters.map(encodeURIComponent)
+        payload.filters = filters.map(encodeURIComponent);
```

Comment:
did prettier here fail on the before already as well?

Reply:
We don't run prettier in CI, so a failure wouldn't be visible if edited in an environment that does not run prettier on file change / save.